src/content/reference/react/useDeferredValue.md

```diff
@@ -32,37 +32,37 @@ function SearchPage() {
 }
 ```
 
-[See more examples below.](#usage)
+[Lihat contoh lainnya di bawah ini.](#usage)
 
 #### Parameters {/*parameters*/}
 
-* `value`: The value you want to defer. It can have any type.
+* `value`: Nilai yang ingin Anda tangguhkan. Nilai ini dapat memiliki tipe apa saja.
 
 #### Returns {/*returns*/}
 
-During the initial render, the returned deferred value will be the same as the value you provided. During updates, React will first attempt a re-render with the old value (so it will return the old value), and then try another re-render in background with the new value (so it will return the updated value). 
+Selama render awal, nilai tangguhan yang dikembalikan akan sama dengan nilai yang Anda berikan. Selama pembaruan, React pertama-tama akan mencoba rerender dengan nilai lama (sehingga akan mengembalikan nilai lama), dan kemudian mencoba rerender lainnya di latar belakang dengan nilai baru (sehingga akan mengembalikan nilai yang diperbarui).
 
 #### Caveats {/*caveats*/}
 
-- The values you pass to `useDeferredValue` should either be primitive values (like strings and numbers) or objects created outside of rendering. If you create a new object during rendering and immediately pass it to `useDeferredValue`, it will be different on every render, causing unnecessary background re-renders.
+- Nilai yang Anda teruskan ke `useDeferredValue` harus berupa nilai primitif (seperti string dan angka) atau objek yang dibuat di luar rendering. Jika Anda membuat objek baru selama perenderan dan langsung meneruskannya ke `useDeferredValue`, objek tersebut akan berbeda di setiap perenderan, menyebabkan rerender latar belakang yang tidak perlu.
```

Comment:
Pass = Oper
```suggestion
- Nilai yang Anda oper ke `useDeferredValue` harus berupa nilai primitif (seperti string dan angka) atau objek yang dibuat di luar rendering. Jika Anda membuat objek baru selama perenderan dan langsung mengopernya ke `useDeferredValue`, objek tersebut akan berbeda di setiap perenderan, menyebabkan *render* ulang latar belakang yang tidak perlu.
```

Reply:
changes delivered on this commit https://github.com/reactjs/id.react.dev/pull/360/commits/8e483b0ea2d00020cce449bdfb354e677e1e652f